vip-cache-manager/js/admin-bar.js

```diff
@@ -0,0 +1,69 @@
+(async () => {
```

Comment:
This javascript file needs more `( nacin spacin )` :)

Reply:
`Prettier` doesn't like  `( nacin spacin )` but let me see what I can do short of fixing each occurrence manually :)

Reply:
Applied `eslint-config-wpvip` to the file (will work on adding it to the repo in a separate PR)